src/components/Login/Login.tsx

```diff
@@ -0,0 +1,104 @@
+import "./Login.css";
+import React, { useState } from "react";
+
+// username: "mor_2314",
+// password: "83r5^_",
+const URL = "https://fakestoreapi.com/auth/login";
+const userData = ["User123", "abcd1234"]
+
+interface LoginProps {
+  loginFlagHandler: (args: boolean) => void;
+}
+
+const Login = ({ loginFlagHandler }: LoginProps) => {
+  const [user, setUser] = useState<string>("");
+  const [pass, setPass] = useState<string>("");
+
+  const changeHandler = (
+    e: React.ChangeEvent<HTMLInputElement>,
+    type: string
+  ) => {
+    if (type === "user") {
+      setUser(e.target.value);
+    } else if (type === "pass") {
+      setPass(e.target.value);
+    }
+  };
+
+  const loginCallHandler = async () => {
+    try {
+      if(!user || !pass) {
+        alert("Your username or password is not correct")
+        return
+      }
+      if(userData.includes(user) && userData.includes(pass)) {
+        sessionStorage.setItem("isAuthorized", "true")
+        loginFlagHandler(true)
+        return
+      }
+      if (user && pass) {
+        const param = {
+          method: "POST",
+          body: JSON.stringify({
+            password: pass,
+            username: user,
+          }),
+        };
+        const response = await fetch(URL, param);
+        const data = await response.json();
+        console.log("data=>>", data);
+        loginFlagHandler(true);
+      }
+    } catch (err) {
+      console.log("Error=>>", err);
+      loginFlagHandler(false);
+    }
+  };
+  return (
+    <div className="overlay" data-testid="login-page">
+      <form
+        data-testid="login-form"
+        className="wrapper"
+        onSubmit={e => {
+          e.preventDefault();
+          loginCallHandler();
+        }}
+      >
+        <h1 data-testid="title" className="title">
+          Sign In
+        </h1>
+        <div className="input_wrapper">
+          <label data-testid="user-label" htmlFor="username">
```

Comment:
`htmlFor` attribute does not work without `id` attribute in the corresponding input element. Please update the code.

Reply:
addressed the comment.